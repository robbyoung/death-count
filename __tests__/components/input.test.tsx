import React from 'react';
import renderer from 'react-test-renderer';
import Input from '../../app/components/input';

describe('Input Component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<Input
            value="Some text"
            onTextChange={() => undefined}
            title='Input Title'
        />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});