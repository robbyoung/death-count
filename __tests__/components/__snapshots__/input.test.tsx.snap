// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Component renders correctly 1`] = `
<View
  style={
    Object {
      "marginLeft": "5%",
      "marginRight": "5%",
      "width": "90%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 18,
        "marginBottom": 5,
        "marginTop": 5,
      }
    }
  >
    Input Title
  </Text>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    rejectResponderTermination={true}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderRadius": 5,
        "color": "#000000",
        "fontSize": 18,
        "padding": 10,
        "width": "100%",
      }
    }
    underlineColorAndroid="transparent"
    value="Some text"
  />
</View>
`;
